AfterShowToDo.md

```diff
@@ -1,6 +1,16 @@
-- [ ] Upload recording to YouTube.
-- [ ] Update README.md with next session.
-- [ ] Send out emails to the authors of the next paper.
-- [ ] Tweet.
-- [ ] Submit to Reddit.
-- [ ] Send the recording to the authors of the current paper.
+Live streaming
+- [ ] Add title and description to the stream.
+- [ ] Copy the public YouTube stream key.
+- [ ] Paste stream key to Jitsi.
+
+Normal recording (not live)
+- [ ] On Anduck's Jitsi server, click start recording.
+- [ ] When the recording stops, the mp4-file will be saved.
+- [ ] Download and edit the episode if needed.
```

Comment:
Why this?

Reply:
It'd require changing the link. If we don't then it's just confusing.

Reply:
Just overall instructions in case we need to use Anduck's server to recording, like for example if the stream doesn't work. 
Primary option would be to live stream from the current [jitsi link](https://meet.jit.si/wasabiresearch). 
Secondary would be to jump in to [Anduck's jitsi](https://jitsi.anduck.net/WasabiResearchExperience) so the mp4 goes to his server, from which I can download it.
Of course, I would stay on both calls to inform about the change of call link, in case someone ends up on a wrong one. And like I said, this is only a backup in case the stream doesn't work. I can remove it too.

Reply:
Actually, secondary options could be to record with OBS on the original jitsi link.
Third option could be to record in different jitsi link.